src/components/Products/Pagination/Pagination.jsx

```diff
@@ -1,16 +1,21 @@
-import { useState } from "react";
+import { useContext, useState } from "react";
+import { ProductsContext } from "../../../ProductsContext";
 
 import './styles.scss';
 
-const Pagination = ({ setCurrentPage, currentPage }) => {
-  const [limitPageNum, setLimitPageNum] = useState(6);
+const Pagination = () => {
+  const productContext = useContext(ProductsContext);
```

Comment:
```suggestion
  const { state, dispatch } = useContext(ProductsContext);
```
Sử dụng destructuring cho cú pháp ngắn gọn hơn nhé